src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -277,6 +277,20 @@ impl ScalarFunc {
         }
         Ok(Some(result))
     }
+
+    #[inline]
+    pub fn date_diff(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
+        let l_t: Cow<Time> = try_opt!(self.children[0].eval_time(ctx, row));
```

Comment:
Is better to name it `lhs`?

Reply:
how about lhs_t and rhs_t?

Reply:
t could be "type" under some context which is confusing 
may I know is there a naming convention the project is following?

Reply:
addressed

Reply:
May I ask what's the `_t` suffix for?